cmd/update-main_test.go

```diff
@@ -82,13 +83,19 @@ func TestReleaseTagToNFromTimeConversion(t *testing.T) {
 func TestDownloadURL(t *testing.T) {
 	minioVersion1 := releaseTimeToReleaseTag(UTCNow())
 	durl := getDownloadURL(minioVersion1)
-	if runtime.GOOS == "windows" {
-		if durl != minioReleaseURL+"minio.exe" {
-			t.Errorf("Expected %s, got %s", minioReleaseURL+"minio.exe", durl)
+	if IsDocker() {
```

Comment:
Why do we do this? There should be no difference between docker and just linux environment.

Reply:
Because Docker env reports a different URL to update than binary only installs @wlan0 for docker updating Minio is 

```
docker pull
```

A binary is to simply 
```
wget <binary...>
```

Reply:
This is just for reporting the update URL then?

Reply:
Yes, that is correct, its a test which was added - I am just letting it pass. We can discuss removing such tests later, as I personally don't like these type of tests.